packages/client/katello-pull-provider-migrate/katello-pull-provider-migrate.spec

```diff
@@ -0,0 +1,43 @@
+Name:           katello-pull-provider-migrate
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        An RPM that migrates katello-agent users to the new pull provider
+BuildArch:      noarch
+
+License:        LGPLv2
+URL:            https://github.com/theforeman/foreman-packaging/packages/client/katello-pull-provider-migrate
```

Comment:
this URL is 404. :)

Reply:
I think I meant
```
https://github.com/theforeman/foreman-packaging/tree/rpm/develop/packages/client/katello-pull-provider-migrate
```
which is a 404 now but won't be when this is merged. I'm not sure what other URL would be correct..

Reply:
https://theforeman.org is a fine URL too ;)

or https://github.com/katello/katello-pull-provider-migrate ;)